### PR TITLE
Clarify that remote SSH access is disabled by default and expand upon…

### PR DESCRIPTION
**Pull request by deddc23efb**
… the details required to set the SSH TCP port field to enable remote SSH access

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @deddc23efb, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;hassio-addons#1594).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: ssh/DOCS.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ You can access the web terminal by clicking the "Open Web UI" button on this add
 
 ### SSH Server Connection
 
-To connect using an SSH client, such as PuTTY, you need to supply additional configuration for this add-on. To enable SSH connectivity, you need to:
+Remote SSH access from the network is disabled by default (See Network below).  To connect using an SSH client, such as PuTTY or Linux terminal, you need to supply additional configuration for this add-on. To enable SSH connectivity, you need to:
```
Comment (frenck) [inline ssh/DOCS.md:34]: Minor tweaks:

```suggestion
Remote SSH access from the network is disabled by default (see Network below). To connect using an SSH client, such as PuTTY or Linux terminal, you need to supply additional configuration for this add-on. To enable SSH connectivity, you need to:
```

Additional concern: 

> To connect using an SSH client, such as PuTTY or Linux terminal

The Linux terminal itself is not an SSH client. The modified section below contains the same error.

File: ssh/DOCS.md [hunk 2]
```diff
@@ -76,7 +76,9 @@ Specifies whether TCP forwarding is permitted or not.
 
 ## Network
 
-This section is only relevant if you want to connect to Home Assistant using an SSH client, such as PuTTY. To enable SSH access via the network, you need to specify which port to use on the Home Assistant host. The number you enter will be used to map that port from the host into the running "Terminal & SSH" add-on. The standard port used for the SSH protocol is `22`.
+This section is only relevant if you want to connect to Home Assistant using an SSH client, such as PuTTY or Linux terminal. To enable SSH remote access from the Network, specify the desired SSH TCP server port in the Network configuration input box. The number you enter will be used to map that port from the host into the running "Terminal & SSH" add-on. The standard port used for the SSH protocol is `22`.  
+
+SSH remote access can be disabled by clearing the input box, saving the config and restarting the add-on.  The default is disabled.
```
Comment (frenck) [inline ssh/DOCS.md:81]: Small tweak to clarify it is turning off again. The section above already tells one how to enable it.

```suggestion
Remote SSH access can be disabled again, by clearing the input box, saving the configuration and restarting the add-on.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @deddc23efb!

I've reviewed it and left some comments. Could you please take a look? Thanks 👍

File: ssh/DOCS.md (hunk 1 above)
Comment (deddc23efb) [reply ssh/DOCS.md:34]: Good point, perhaps substitute "PuTTY or ssh from a shell command" ...  could even drop that change if it's not adding value.

**Review by pvizeli — APPROVED** [review]

**Pull request merged**